app/jobs/notifications_reader_job.rb

```diff
@@ -9,6 +9,11 @@ def perform(commentable_id:, commentable_type:, user_id:)
         where(recipient_id: user_id).
         mark_all_as_read!
 
+      notifications_by_card(id: commentable_id).
```

Comment:
I added a separate call because the query is too tangled within commentable, and the action of assigning users to a card only creates a notification but does not create a comment. I'm not sure what's the best way of moving forward. Maybe we can uncouple notifiable and commentable but I'm afraid there will be much more work to be done and might introduce braking changes.